lib/MusicBrainz/Server/Controller/Doc.pm

```diff
@@ -26,24 +27,31 @@ sub show : Path('')
         return;
     }
 
-    my $bare = $c->req->param('bare') || 0;
-    $c->stash(
-        id => $id,
-        page => $page,
-        google_custom_search => DBDefs->GOOGLE_CUSTOM_SEARCH,
-    );
-
     if ($id =~ /^[^:]+:/i && $id !~ /^Category:/i) {
         $c->response->redirect(sprintf('http://%s/%s', DBDefs->WIKITRANS_SERVER, $id));
         $c->detach;
     }
 
+    my %props = (
+        useGoogleCustomSearch   => boolean_to_json(DBDefs->GOOGLE_CUSTOM_SEARCH),
```

Comment:
We don't care about the Google Custom Search token here, just about whether it exists. As such, it seems better IMO to pass a boolean to JSON rather than the actual token.

Reply:
That makes sense. Though `GOOGLE_CUSTOM_SEARCH` is already in DBDefs.js - I'd think you can just import and perform this check in JS rather than passing an extra prop.